src/app/access-control/epeople-registry/eperson-form/eperson-form.component.ts

```diff
@@ -591,6 +591,38 @@ export class EPersonFormComponent implements OnInit, OnDestroy {
     }
   }
 
+  /**
+   * Deletes a given group from the Group list of the eperson currently being edited present in
+   * @param group  group we want to delete as of which the current eperson being edited is member of
+   */
+  deleteGroupFromMember(group: Group) {
```

Comment:
This method is only being called in your spec file.  I suspect it's supposed to be called by an actual delete button.